app/controllers/api_controller/chargeback.rb

```diff
@@ -0,0 +1,45 @@
+class ApiController
+  module Chargeback
+    # Request
```

Comment:
I think the right place for examples of these requests would be either in the specs or documentation, would you mind losing these comments?